serialize/decoder.go

```diff
@@ -111,12 +111,18 @@ func (d *decoder) decodeTag() (ident.Tag, error) {
 	if err != nil {
 		return ident.Tag{}, err
 	}
+	if len(name.Bytes()) == 0 {
+		return ident.Tag{}, errEmptyTagNameLiteral
```

Comment:
Can you wrap this in an invalid params error (to return Bad Request)?  `xerrors.NewInvalidParamsError(..)`?